### PR TITLE
Install and initialize health_check gem

### LINKED ISSUE #367: Rails Health Check
https://github.com/ianheggie/health_check

### PR DESCRIPTION
**Pull request by dmitrytrager**
closes #367

### CODE REVIEW

**Comment by dmitrytrager** [comment]
review needed

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ gem "responders"
 gem "rollbar"
 gem "seedbank"
 gem "stackprof"
+gem "health_check"
```
Comment (timurvafin) [inline Gemfile]: Could you please make sure gems sorted alphabetically?

File: config/initializers/health_check.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  # Set to 200 if you want your want to distinguish between partial (healthy property == false) and
+  # total failure of rails application (http status of 500 etc)
+
+  config.http_status_for_error_object = 500
+
+  # You can customize which checks happen on a standard health check
+  config.standard_checks = %w(database migrations custom)
+
+  # You can set what tests are run with the 'full' or 'all' parameter
+  config.full_checks = %w(database migrations custom email cache)
```
Comment (timurvafin) [inline config/initializers/health_check.rb]: What does `custom` check do in `rails-base`?

**Comment by dmitrytrager** [comment]
@timurvafin updated

**Comment by dmitrytrager** [comment]
@timurvafin  updated

**Comment by timurvafin** [comment]
@dmitrytrager looks ok to me except CHANGE log notes.

**Pull request merged**